backends/cadence/hifi/operators/op_add.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <executorch/kernels/portable/cpu/scalar_utils.h>
+#include <executorch/kernels/portable/cpu/util/broadcast_util.h>
+#include <executorch/kernels/portable/cpu/util/functional_util.h>
+#include <executorch/runtime/kernel/kernel_includes.h>
+#include <executorch/runtime/platform/assert.h>
+#include "kernels.h"
+
+namespace torch {
+namespace executor {
+namespace native {
```

Comment:
I think this should be the `impl::hifi` namespace (whichever one we use for the custom ops), since we re-implement those and they're not torch owned anymore. The `yaml` file would need to be modified to reflect that too.

Reply:
As we are keeping native code as well, we do have dependency on name spaces mainly related to utility. Should we add them like follows and use impl::hifi ?
_using executorch::runtime::CppTypeToScalarType;
using executorch::runtime::can_cast;
using torch::executor::Error;
using Scalar = exec_aten::Scalar;_   
etc..

Reply:
Yes that's a good point, I think we should probably do that. We can possibly have a header file where we import all of those too. @hsharma35 any thoughts?

Reply:
Let's keep using-declarations to cpp only.